homeassistant/components/clickatell/notify.py

```diff
@@ -25,27 +28,37 @@
 )
 
 
-def get_service(
+async def async_get_service(
     hass: HomeAssistant,
     config: ConfigType,
     discovery_info: DiscoveryInfoType | None = None,
 ) -> ClickatellNotificationService:
     """Get the Clickatell notification service."""
-    return ClickatellNotificationService(config)
+    key = config.get(CONF_API_KEY)
+    url = f"{BASE_API_URL}{key}"
```

Comment:
I am confused by you added key and url the ClickatellNotificationService initialiser. You do not seem to use them